npm/css-src/sass/_drag-and-drop.scss

```diff
@@ -2,31 +2,53 @@
   margin-bottom: 40px;
 }
 
+.drag-and-drop__button {
+  margin-bottom: 32px;
+  margin-top: 65px;
+  padding-left: 15px;
+  position: static;
+  width: auto;
+}
+
+.drag-and-drop__button::before {
+  position: static;
+}
+
+.drag-and-drop__button--highlight {
+  background-color: $govuk-focus-colour;
+  box-shadow: 0 -2px $govuk-focus-colour, 0 4px govuk-colour('black');
+  color: $govuk-text-colour;
+  outline: none;
+}
+
+.drag-and-drop--dragover {
```

Comment:
It looks like this class goes on the `.drag-and-drop__dropzone` element, which means it should be `.drag-and-drop__dropzone--dragover` to be consistent with BEM.

Reply:
Ah, yes! Sure, will do!